js/testing/testing.ts

```diff
@@ -71,25 +74,21 @@ async function runTests() {
   console.log("running", tests.length, "tests");
   for (let i = 0; i < tests.length; i++) {
     const { fn, name } = tests[i];
-    let result = green_ok();
-    console.log("test", name);
     try {
       await fn();
+      console.log("test", name, "...", green_ok());
```

Comment:
Could this instead first write to stdout instead of console.log (so as to avoid the newline)?

Reply:
@kt3k That is likely because [TTYs are line buffered by default](https://eklitzke.org/stdout-buffering). To do this we probably need call [sync(2)](http://man7.org/linux/man-pages/man2/sync.2.html) or set the TTY in unbuffered mode.

Maybe @piscisaureus remembers how we did this in Node...

Reply:
Upon further investigation, we are using tokio-fs's stdout, which writes to [std::io::Stdout](https://doc.rust-lang.org/std/io/struct.Stdout.html), which has a flush method (suggesting it is line buffered). If we were issuing a write syscall directly, this wouldn't happen. 
We should probably expose that flush method as `deno.stdout.flush()`...

Reply:
I've opened an issue for this: #948